configs/postcss.config.js

```diff
@@ -1,7 +1,6 @@
 /* eslint-disable global-require, import/no-commonjs, import/unambiguous */
 module.exports = {
     plugins: [
-        require('postcss-flexbox-unboxer')(),
```

Comment:
Was that added in this PR on purpose?

Reply:
Yes it's required otherwise the `-webkit-box` is removed and we can't do multiline truncate. Also it has be removed from the product too a while ago